libs/logger/src/logger.cpp

```diff
@@ -25,7 +25,7 @@ PRAGMA_CLANG_WARNING_POP
 
 tfc::logger::logger::logger(std::string_view key) : key_{ key } {
   // Create sinks
-  std::vector<spdlog::sink_ptr> sinks{ std::make_shared<spdlog::sinks::tfc_systemd_sink_mt>(key_) };
+  std::vector<spdlog::sink_ptr> sinks{ std::make_shared<spdlog::sinks::stderr_color_sink_mt>() };
```

Comment:
This propably double logs to the console when --stdout is enabled.

Reply:
removed double logs, this will need to be reverted